src/lib/inputs/Input.svelte

```diff
@@ -103,7 +111,15 @@
     {disabled}
     {...$$restProps}
     style={inputStyle}
+    readonly={readOnly}
```

Comment:
for accessibility you _could_ consider doing readOnly AS your disabled state and then adding aria-disabled={readonly}. When you use the native 'disabled' global html attr it completely eliminates the input from tab order and screen readers completely omit announcing it. but more of a user experience thing than a 'code is wrong' thing

Reply:
INTERESTING! I'll try this out. Do you think this is something that could be useful on all elements? Or more-so just for inputs?

Reply:
If you refer to line 56, readOnly's state is altered if any of the loading, disabled, or error states are true. Which is why, in this case, readOnly is set to the readonly property. Would it work just as well to just add aria-disabled={readOnly}?